src/components/ha-label-badge.html

```diff
@@ -52,10 +52,11 @@
       border-radius: 1em;
       padding: 8% 16%;
       font-weight: 500;
-      text-transform: uppercase;
       overflow: hidden;
+      text-transform: uppercase;
       text-overflow: ellipsis;
       transition: background-color .3s ease-in-out;
+      text-transform: var(--ha-label-badge-label-text-transform);
```

Comment:
This should have a fallback of `uppercase`, like it used to be. Otherwise you change the style of this component from working without state label badge

Reply:
yeah, thanks for catching that!